index.js

```diff
@@ -35,7 +37,18 @@ const buildPostCache = async () => {
 
         posts.sort((l, r) => r.timestamp - l.timestamp);
 
+        
+        for (const p of posts) {
+            postsAlt.push({
+                title: p.title,
+                author: p.author,
+                text: p.descriptionText,
```

Comment:
why only the first 24 words? ideally it indexes the entire post, no? perhaps we can use the `content` field on post since that should contain the entire text

Reply:
I mainly did this because of how big the payload would be, and also chances are what an article is about is mentioned within the first little bit. I could change this though.